src/components/marker/marker.helper.js

```diff
@@ -93,4 +101,28 @@ function _memoizedComputeMarkerId() {
  */
 const getMarkerId = _memoizedComputeMarkerId();
 
-export { getMarkerId };
+/**
+ * Computes the three marker sizes
+ *
```

Comment:
Sorry about the delay, could you just please write down a comment in this description pointing out that we will return size `0`, for shapes that are supported in our `getNormalizedNodeCoordinates` in order to get a better/more accurate integration with the node and arrow for directional graphs? Anything else who would add as well?

Thank you! It looks very good now :+1:

Reply:
Nothing else to add from my side. 
Don't hesitate if you have anything more to add, I will change it :smiley: